.snyk

```diff
@@ -3,6 +3,7 @@
 # https://docs.snyk.io/snyk-cli/commands/ignore
 exclude:
   global:
+    - internal/tools/update-readme/main.go
```

Comment:
See here: https://github.com/openshift/openshift-mcp-server/pull/41#issuecomment-3375710107

Since it is an internal tool with existing path sanitization (filepath.Clean and filepath.Localize on line 31-32). The warnings are likely false positives...

Reply:
Nit: That would be better to add a justification about the exclusion but not a blocker.